server/core/region.go

```diff
@@ -406,6 +413,30 @@ func (r *RegionInfo) GetStat() *pdpb.RegionStat {
 	}
 }
 
+// UpdateBuckets sets the buckets of the region.
+func (r *RegionInfo) UpdateBuckets(buckets *metapb.Buckets) bool {
+	// bucket is only nil in test cases.
```

Comment:
The bucket shouldn't be nil except in the test cases?

Reply:
fix